bitcoin/rpc.py

```diff
@@ -553,111 +557,931 @@ def getrawmempool(self, verbose=False):
+        #Works for both addresses and pubkeys, but hex() vs str() is annoying.
+        #TODO see if CPubKey.__str__() is used elsewhere or can be changed.
+        if isinstance(keys[0], CBitcoinAddress): 
+            keys = [str(k) for k in keys]
+        elif isinstance(keys[0], (CPubKey, bytes)):  
+            keys = [k.hex() for k in keys]
+        r = self._call('addmultisigaddress', nrequired, keys, label, address_type)
+        r['address'] = CBitcoinAddress(r['address'])
+        r['redeemScript'] = CScript.fromhex(r['redeemScript'])
+        return r
+
+    def backupwallet(self, destination):
+        """copies current wallet file to destination
+        destination - path to directory with or without filename
+        """
+        self._call('backupwallet', destination)
+
+    def bumpfee(self, txid, options=None):
+        """Bump fee of transation in mempool"""
+        if not isinstance(txid, str):
+            txid = b2lx(txid)
+        return self._call('bumpfee', txid, options)
+
+    def createwallet(self, wallet_name, disable_priv_keys=None, blank=None, passphrase=None, avoid_reuse=None ):
+        """Create a new Wallet 
+        wallet_name - name
+        disable_priv_keys - watch_only, default=False
+        blank - create a blank wallet with no seed or keys
+        passphrase - encrypt wallet with passphrase
+        avoid_reuse - segregate reused and clean coins. Better privacy
+        Return a JSON object about the new wallet
+        """
+        return self._call('createwallet', wallet_name, disable_priv_keys, blank, passphrase, avoid_reuse)
+
+    def dumpprivkey(self, addr):
+        """Return the private key matching an address
+        """
+        r = self._call('dumpprivkey', str(addr))
+        return CBitcoinSecret(r)
+    
+    def dumpwallet(self, filename):
+        """Dump all wallet keys and imported keys to a file.
+        NO OVERWRITING ALLOWED
+        returns a JSON object with full absolute path
+        """
+        self._call('dumpwallet', filename)
+
+    def encryptwallet(self, passphrase):
+        """
+        Encrypts wallet for the first time.
+        This passphrase will be required for all signing after call.
+        """
+        self._call('encryptwallet', passphrase)
+
+    def getaddressesbylabel(self, label):
+        """Return a JSON object with addresses as keys"""
+        # Convert to CBitcoinAddress? 
+        # not converting addresses makes the dict searchable.
+        return self._call('getaddressbylabel', label)
+
+    def getaccountaddress(self, account=None):
+        """Return the current Bitcoin address for receiving payments to this
+        account."""
+        r = self._call('getaccountaddress', account)
+        return CBitcoinAddress(r)
+
+    def getaddressinfo(self, address):
+        """Return a JSON object of info about address"""
+        address = str(address)
+        r = self._call('getaddressinfo', address)
+        if r['script'] == 'scripthash':
```

Comment:
This fails with `KeyError: 'script'` because `script` is only optional. I would recommend to extend the check (e.g. `if r['isscript'] and r['script'] == 'scripthash':`)

Reply:
Thank you for testing and reviewing! I will fix this asap.

Reply:
Fixing now. Sorry for the delay. Thank you for reviewing!